clients/gtest/CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ target_include_directories( hipsolver-test
   SYSTEM PRIVATE
     $<BUILD_INTERFACE:${CBLAS_INCLUDE_DIRS}>
     $<BUILD_INTERFACE:${HIP_INCLUDE_DIRS}>
-    ${ROCM_PATH}/hsa/include
+    ${ROCM_PATH}/include
```

Comment:
Can we just delete `${ROCM_PATH}/hsa/include` without adding anything? All of our include paths should be coming from targets we link.